code/datums/spells/devil.dm

```diff
@@ -35,6 +35,12 @@
 /obj/effect/proc_holder/spell/targeted/summon_pitchfork/ascended
 	pitchfork_type = /obj/item/weapon/twohanded/pitchfork/demonic/ascended
 
+/obj/effect/proc_holder/spell/targeted/summon_pitchfork/violin
```

Comment:
for a moment i though you were making the gold violin a subtype of pitchfork weapons